src/shaderlib/picking/shadertools-picking.md

```diff
@@ -0,0 +1,77 @@
+# picking (Shader Module)
+
+Provides support for color-coding-based picking and highlighting. In particular, supports picking a specific instance in an instanced draw call and highlighting an instance based on its picking color.
```

Comment:
**DOC CHANGE**: ..., and correspondingly, supports picking and highlighting groups of primitives with the same picking color in non-instanced draw-calls.